package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@roadiehq/backstage-plugin-github-pull-requests",
-  "version": "0.5.2",
+  "version": "0.5.3",
```

Comment:
thanks so much for doing this!  🎉 🎉  would Semver-aligned upgrade to add GHE support (minor, backwards compatible) make this a 0.6.0?

Reply:
I was thinking if those changes  should be marked as minor or patch but you are right. Lets make it 0.6.0 👍